README.md
<p align="center"><img src="https://dl.ascoos.com/images/ascoos.png" height=120 /></p>

<hr />

# popular_content


## Description

This block displays a box with the most popular contents of the website.

The contents can be displayed in the current or in all languages where texts are available.

<p align="center"><img src="https://dl.ascoos.com/cms/ascoos/ext/blocks/popular-content/images/screenshot.png" height=640 /></p>

***

<br>

## Management

The block is managed through the ASCOOS Explorer program

<p align="center"><img src="https://dl.ascoos.com/cms/ascoos/ext/blocks/popular-content/images/ASCOOS_Edit_Block_p640.png" height=640 /></p>

<br>

We can configure its contents from the parameters

<p align="center"><img src="https://dl.ascoos.com/cms/ascoos/ext/blocks/popular-content/images/ASCOOS_Edit_Block_Parameters_p640.png" height=640 /></p>

***

<br>

## Documentation

You can find more information about the block on the documentation page of [Ascoos CMS - BLOCKS - Popular Content Documantation](https://docs.ascoos.com/ascoos/ext/blocks/popular-content)

<br>

## Feedback

Please send any feedback or suggestions to [@ascoos](https://twitter.com/ascoos) (Twitter) or [create an issue](https://github.com/ascoos/popular_content/issues) on GitHub.

<br> 
 
## License

[AGL (ASCOOS General License)](http://docs.ascoos.com/lics/ascoos/AGL.html)

<br>

***

<br>

## Download

- [ASCOOS CMS Popular Content from Original Download site](https://dl.ascoos.com/cms/ascoos/ext/blocks/popular-content/popular_content-v010000.zip)

- [ASCOOS CMS Popular Content from Github](https://github.com/ascoos/popular_content/releases)


CHANGELOG.md
## [1.0.1] - 2023-06-29

- Change file install.php

***

## [1.0.0] - 2012-07-01

- Initial release

index.php
<?php
/*
  __ _  ___  ___ ___   ___   ___     ____ _ __ ___   ___
 / _` |/  / / __/ _ \ / _ \ /  /    / __/| '_ ` _ \ /  /
| (_| |\  \| (_| (_) | (_) |\  \   | (__ | | | | | |\  \
 \__,_|/__/ \___\___/ \___/ /__/    \___\|_| |_| |_|/__/
                                                                  
*************************************************************************************
* @ Project ASCOOS                                                                  *
* @ Copyright (C) 2007 - 2012 AlexSoft Software.                                    *
* @ Address: Konstantinoupoleos 88, GR 68100, Alexandroupolis, Evros, Greece        *
* @ Tel: +30 2551 031999                                                            *
* @ Creator: Drogidis Christos                                                      *
* @ ASCOOS CMS Site: www.ascoos.com                                                 *
* @ Creator Site: www.alexsoft.gr                                                   *
* @ emails: webmaster@ascoos.com, webmaster@alexsoft.gr                             *
* @ license site: http://www.alexsoft.gr/licence/ascoos/index.php                   *
* @ Copyrighted Commercial Software                                                 *
* @ Program ASCOOS CMS Manager                                                      *
*************************************************************************************

* @ Package           : ASCOOS CMS - Frontend
* @ Subpackage        : Block Manager - Popular Content 
* @ ASCOOS Version    : Lite - 1.0.0
* @ File Name         : /blocks/popular_content/index.php
* @ File No.          : 5 - $release: 1.0 - $revision: 0 - $build: 0
* @ Created Date      : 2012-07-01 20:00:00 UTC+2 
* @ Updated Date      : 
* @ Author            : Drogidis Christos
* @ Author email      : webmaster@alexsoft.gr
* @ Author website    : www.alexsoft.gr
*/

defined ("ALEXSOFT_RUN_CMS") or die("Prohibition of Access.");

global $cms_site, $objDatabase, $ASCOOS, $my, $objDual;

// Get Value Block Parameters
$count 			= $block->getParam('int', 'count', 5 );
$all_lang		= $block->getParam('bool', 'all_lang', false );
$show_hits		= $block->getParam('bool', 'show_hits', false );
$type 			= $block->getParam('lstr', 'type', '');
$cat_ids 		= $block->getParam('lstr', 'cat_ids', '' );
$except_cat_ids = $block->getParam('lstr', 'except_cat_ids', '' );
$theme		 	= $block->getParam('lstr', 'theme', 'default' );

// load Block Theme
$block->loadTheme($theme);

$where = array();

if (!$all_lang) $where[] = "a.lang_id = ".$ASCOOS['lang']->id;
if ($type != '') $where[] = "a.type IN (".$type.")";
if ($cat_ids != '') $where[] = "a.cat_id IN (".$cat_ids.")";
if ($except_cat_ids != '') $where[] = "a.cat_id NOT IN (".$except_cat_ids.")";
$where[] = "a.published=1";
$where[] = "a.groupid <= ".$my->groupid;

$query = "SELECT a.id, a.article_id, a.title, a.lang_id, a.cat_id, a.created, a.hits, a.access, a.groupid, l.domain AS flag"
	. "\nFROM #__articles AS a"
	. "\n LEFT JOIN #__languages AS l ON l.id = a.lang_id"
	. (count( $where ) ? "\nWHERE " . implode( ' AND ', $where ) : "")
	. "\nORDER BY a.hits DESC"
	. "\nLIMIT ".$count;
$objDatabase->setSQLQuery( $query );
$rows = $objDatabase->getObjects();

unset($where);

if (count($rows)) {
	$text = '';
	$text .= "<div class=\"block-popular-content-".$theme."\">";
	if ($block->getVar('show_title')) { 
		$text .= "<div class=\"header\"><h3>".$block->getTitle()."</h3></div><div class=\"clear\"></div>";
	}
	$text .= "<div class=\"text\"><div class=\"table\">";
	foreach ( $rows as $row ) { 
		// If you do not have the user permissions to read the article $row, then dodging article.
		if (!$objDual->checkAccess($row)) {
			continue;
		} else { // .... else view article link
	       	$text .= "<div class=\"row\">";
			if ($all_lang) $text .= "<div class=\"cell\"><img src=\"".$cms_site."/images/kernel/flags/".$row->flag.".png\" alt=\"".$row->title."\" border=\"0\" /></div>";
    		$text .= "<div class=\"cell\"><a href=\"".asc2seo('index.php?p=articles&amp;t=view&amp;id='.$row->article_id)."\">".$row->title."</a></div>";
			if ($show_hits) $text .= "<div class=\"cell right\">".$row->hits."</div>";
    	    $text .= "</div>";
		}
    }
    $text .= "</div></div></div>";
	echo $text;
	unset($text);
	unset($rows);
}
?>

params.php
<?php
/*
  __ _  ___  ___ ___   ___   ___     ____ _ __ ___   ___
 / _` |/  / / __/ _ \ / _ \ /  /    / __/| '_ ` _ \ /  /
| (_| |\  \| (_| (_) | (_) |\  \   | (__ | | | | | |\  \
 \__,_|/__/ \___\___/ \___/ /__/    \___\|_| |_| |_|/__/
                                                                  
*************************************************************************************
* @ Project ASCOOS                                                                  *
* @ Copyright (C) 2007 - 2012 AlexSoft Software.                                    *
* @ Address: Konstantinoupoleos 88, GR 68100, Alexandroupolis, Evros, Greece        *
* @ Tel: +30 2551 031999                                                            *
* @ Creator: Drogidis Christos                                                      *
* @ ASCOOS CMS Site: www.ascoos.com                                                 *
* @ Creator Site: www.alexsoft.gr                                                   *
* @ emails: webmaster@ascoos.com, webmaster@alexsoft.gr                             *
* @ license site: http://www.alexsoft.gr/licence/ascoos/index.php                   *
* @ Copyrighted Commercial Software                                                 *
* @ Program ASCOOS CMS Manager                                                      *
*************************************************************************************

* @ Package           : ASCOOS CMS - Frontend
* @ Subpackage        : Block Manager - Popular Content 
* @ ASCOOS Version    : Lite - 1.0.0
* @ File Name         : /blocks/popular_content/index.php
* @ File No.          : 6 - $release: 1.0 - $revision: 0 - $build: 0
* @ Created Date      : 2012-07-01 20:00:00 UTC+2 
* @ Updated Date      : 
* @ Author            : Drogidis Christos
* @ Author email      : webmaster@alexsoft.gr
* @ Author website    : www.alexsoft.gr
*/

defined ("ALEXSOFT_RUN_CMS") or die("Prohibition of Access.");

global $ASCOOS, $apps_path;

require_once($apps_path."/articles/languages/".$ASCOOS['alang']->filename.".php");
$lngArticleTypes = new TArticleLanguage;

function getCategories()
{
	global $ASCOOS, $my, $objDatabase;

	$where = array();
	$where[] = "owner = 'articles'";
	$where[] = "groupid <= ".$my->groupid;
  	$where[] = "published=1";

	$query = "SELECT cat_id, title, access, groupid"
		. "\nFROM #__categories"
	   	. (count( $where ) ? "\nWHERE " . implode( ' AND ', $where ) : "")
		. "\nORDER BY cat_id ASC"
	;

	$objDatabase->setSQLQuery( $query );
	$rows = $objDatabase->getObjects();
	unset($where);

	$arr = array();
	foreach ($rows as $row)	$arr[$row->cat_id] = ascoos_langCorrectItem( $row->title, 'topic', true );

	unset($rows);
	return $arr;
}

$ASCOOS['extraParamFields']['ArticleTypes'] = $lngArticleTypes->article_types;
$cat = getCategories();
$ASCOOS['extraParamFields']['ArticleCategories'] = $cat;
$ASCOOS['extraParamFields']['ArticleExceptCategories'] = $cat;
?>

install.php
<?php
/*
  __ _  ___  ___ ___   ___   ___     ____ _ __ ___   ___
 / _` |/  / / __/ _ \ / _ \ /  /    / __/| '_ ` _ \ /  /
| (_| |\  \| (_| (_) | (_) |\  \   | (__ | | | | | |\  \
 \__,_|/__/ \___\___/ \___/ /__/    \___\|_| |_| |_|/__/
                                                                  
*************************************************************************************
* @ Project ASCOOS                                                                  *
* @ Copyright (C) 2007 - 2012 AlexSoft Software.                                    *
* @ Address: Konstantinoupoleos 88, GR 68100, Alexandroupolis, Evros, Greece        *
* @ Tel: +30 2551 031999                                                            *
* @ Creator: Drogidis Christos                                                      *
* @ ASCOOS CMS Site: www.ascoos.com                                                 *
* @ Creator Site: www.alexsoft.gr                                                   *
* @ emails: webmaster@ascoos.com, webmaster@alexsoft.gr                             *
* @ license site: http://www.alexsoft.gr/licence/ascoos/index.php                   *
* @ Copyrighted Commercial Software                                                 *
* @ Program ASCOOS CMS Manager                                                      *
*************************************************************************************

* @ Package           : ASCOOS CMS - Frontend
* @ Subpackage        : Block Manager - Popular Content 
* @ ASCOOS Version    : Lite - 1.0.0
* @ File Name         : /blocks/popular_content/install.php
* @ File No.          : 1 - $release: 1.0 - $revision: 0 - $build: 0
* @ Created Date      : 2012-07-01 20:00:00 UTC+2 
* @ Updated Date      : 
* @ Author            : Drogidis Christos
* @ Author email      : webmaster@alexsoft.gr
* @ Author website    : www.alexsoft.gr
*/

defined ("ALEXSOFT_RUN_CMS") or die("Prohibition of Access.");

global $objInstaller;

// Adjust the installer to work at BLOCK and give the block that will handle
$objInstaller->setExtension('popular_content');

// We adjust the installer so that after installation, to take us or not part of the Block configuration.
$objInstaller->afterSetParams(false);

// If the block is not installed then run the installation.
if (!$objInstaller->isInstalled()) {
	// We create the path of the Block.
	$objInstaller->createPath();
	
	// If the installation files on the Server is successful
	if ( $objInstaller->extractPackage() )
	{
		// We export themes in internal path "fronts". !!!! Not Change for Blocks. !!!!
		$objInstaller->extractThemes('fronts');
		
		// We take the position id, show called "NOTHING".
		$block_pos = $objInstaller->getPosition('NOTHING');

		// We take the sort that will apply to the position.
		$block_order = $objInstaller->getOrderPosition('NOTHING');
	
		// Place the Block in the database.
		$objInstaller->addSQL("INSERT INTO #__blocks VALUES(NULL, '".$objInstaller->name."', '".$objInstaller->name."', '0', '0', '{\"en\":\"Popular\",\"el\":\"Δημοφιλέστερα\"}', '', '', '', '', ".$block_pos.", ".$block_order.", '', '0', '0', '0', '0', '{\"count\":5,\"all_lang\":0,\"show_hits\":1,\"theme\":\"cleargray\",\"type\":1}');");
		
		// pour the settings from the installer.
		$objInstaller->clear();
		
	} else {
		// else otherwise cancel the installation and pour the settings from the installer.
		$objInstaller->cancelInstallation();
	}
} else { // else pour the settings from the installer.
	$objInstaller->clear();
}
?>

languages/greek.php
<?php
/*
  __ _  ___  ___ ___   ___   ___     ____ _ __ ___   ___
 / _` |/  / / __/ _ \ / _ \ /  /    / __/| '_ ` _ \ /  /
| (_| |\  \| (_| (_) | (_) |\  \   | (__ | | | | | |\  \
 \__,_|/__/ \___\___/ \___/ /__/    \___\|_| |_| |_|/__/
                                                                  
*************************************************************************************
* @ Project ASCOOS                                                                  *
* @ Copyright (C) 2007 - 2012 AlexSoft Software.                                    *
* @ Address: Konstantinoupoleos 88, GR 68100, Alexandroupolis, Evros, Greece        *
* @ Tel: +30 2551 031999                                                            *
* @ Creator: Drogidis Christos                                                      *
* @ ASCOOS CMS Site: www.ascoos.com                                                 *
* @ Creator Site: www.alexsoft.gr                                                   *
* @ emails: webmaster@ascoos.com, webmaster@alexsoft.gr                             *
* @ license site: http://www.alexsoft.gr/licence/ascoos/index.php                   *
* @ Copyrighted Commercial Software                                                 *
* @ Program ASCOOS CMS Manager                                                      *
*************************************************************************************

* @ Package            : ASCOOS CMS - Frontend
* @ Subpackage         : Block Manager - Popular Contents 
* @ ASCOOS Version     : Lite - 1.0.0
* @ File Name          : /blocks/popular_conent/languages/greek.php
* @ File No.           : 7 - $release: 1.0 - $revision: 0 - $build: 0
* @ Created Date       : 2012-07-01 20:00:00 UTC+2 
* @ Updated Date       : 
* @ Author             : Drogidis Christos
* @ Author email       : webmaster@alexsoft.gr
* @ Author website     : www.alexsoft.gr
* @ Translator         :
* @ Translator email   : 
* @ Translator website : 
* @ Translate Date     : 
*/

defined ("ALEXSOFT_RUN_CMS") or die("Prohibition of Access.");

class TBlock_popular_content_Language extends TObject
{
	public $blockname = "Δημοφιλέστερα Περιεχόμενα";

	public $APL_license = "Ascoos General License (AGL)";
	public $APL_author = "ΔΡΟΓΚΙΔΗΣ ΧΡΗΣΤΟΣ";
	public $APL_creation = "";
	public $APL_copyright = "ALEXSOFT ΛΟΓΙΣΜΙΚΟ";
	public $APL_translator = "";
	public $APL_translatorEmail = "";
	public $APL_translatorUrl = "";
	public $APL_desc = "Εμφανίζει ένα πλαίσιο που περιέχει τα δημοφιλέστερα περιεχόμενα της ιστοσελίδας.";

	public $APL_paramgroup_general_label = "▼ &nbsp; Επιλογές Γενικών Παραμέτρων";
	public $APL_paramgroup_general_label_info = "<strong>ΕΠΙΛΟΓΕΣ ΓΕΝΙΚΩΝ ΠΑΡΑΜΕΤΡΩΝ</strong><br />--------------------------------------<br /><br />Στο πλαίσιο αυτό μπορείτε να επιλέξετε από διάφορες γενικές παραμέτρους και να διαμορφώσετε δυναμικά την ενότητα";
	public $APL_paramgroup_categories_label = "▼ &nbsp; Επιλογές Κατηγοριών";
	public $APL_paramgroup_categories_label_info = "<strong>ΕΠΙΛΟΓΕΣ ΚΑΤΗΓΟΡΙΩΝ</strong><br />--------------------------------------<br /><br />Στο πλαίσιο αυτό μπορείτε να διαμορφώσετε τις παραμέτρους προέλευσης των περιεχομένων της ενότητας";
	public $APL_count_label = "Αριθμός άρθρων";
	public $APL_count_desc = "<strong>ΑΡΙΘΜΟΣ ΑΡΘΡΩΝ</strong><br />--------------------------------------<br /><br />Εδώ πρέπει να επιλέξετε τον αριθμό των άρθρων που θα εμφανιστούν";
	public $APL_all_lang_label = "Εμφάνιση ανεξαρτήτως γλώσσας";
	public $APL_all_lang_desc = "<strong>ΕΜΦΑΝΙΣΗ ΑΝΕΞΑΡΤΗΤΑ ΑΠΟ ΓΛΩΣΣΑ ΣΥΝΤΑΞΗΣ</strong><br />--------------------------------------<br /><br />Θέλετε να περιληφθούν όλα τα άρθρα, ανεξάρτητα από την γλώσσα που συντάχθηκαν;<br /><br />Αυτό θα έχει ως αποτέλεσμα να εμφανιστούν όλα τα άρθρα ανεξάρτητα από την γλώσσα σύνταξης τους.<br /><br />Για καλύτερη ενημέρωση στα αριστερά θα εμφανίζεται μια σημαία για να προσδιορίζει την γλώσσα σύνταξης.";
	public $APL_show_hits_label = "Εμφάνιση Προβολών";
	public $APL_show_hits_desc = "<strong>ΕΜΦΑΝΙΣΗ ΠΡΟΒΟΛΩΝ</strong><br />--------------------------------------<br /><br />Θέλετε να εμφανίζεται στήλη με τις συνολικές προβολές του περιεχομένου;";
	public $APL_theme_label = "Θέμα εμφάνισης Ενότητας";
	public $APL_theme_placeholder = "Επιλέξτε θέμα εμφάνισης της ενότητας";
	public $APL_theme_desc = "<strong>ΘΕΜΑ ΕΜΦΑΝΙΣΗΣ ΕΝΟΤΗΤΑΣ</strong><br />--------------------------------------<br /><br />Επιλέξτε το θέμα εμφάνισης που θα υλοποιηθεί η ενότητα &laquo;%s&raquo;.<br /><br />Κάθε ενότητα έχει τουλάχιστο το προκαθορισμένο θέμα εμφάνισης με την ονομασία &laquo;Default&raquo;";
	
	public $APL_type_label = "Τύποι άρθρων";
	public $APL_type_placeholder = "Επιλέξτε τύπους άρθρων";
	public $APL_type_desc = "<strong>ΤΥΠΟΙ ΑΡΘΡΩΝ</strong><br />--------------------------------------<br /><br />Ποιοι τύποι άρθρων θέλετε να συμπεριληφθούν; Αυτό θα έχει ως αποτέλεσμα την εμφάνιση μόνο των άρθρων που έχουν ώς τύπο έναν από τους οριζόμενους.";
	public $APL_cat_ids_label = "Κατηγορίες που θα περιληφθούν";
	public $APL_cat_ids_placeholder = "Επιλέξτε Κατηγορίες Άρθρων";
	public $APL_cat_ids_desc = "<strong>ΚΑΤΗΓΟΡΙΕΣ ΠΟΥ ΘΑ ΠΕΡΙΛΗΦΘΟΥΝ</strong><br />--------------------------------------<br /><br />Ποιες κατηγορίες θέλετε να περιληφθούν;<br /><br />Αυτό θα έχει ως αποτέλεσμα μόνο τα άρθρα που έχουν μια από τις κατηγορίες αυτές να εμφανιστούν στην λίστα των πρόσφατων άρθρων.";
	public $APL_except_cat_ids_label = "Κατηγορίες που θα εξαιρεθούν";
	public $APL_except_cat_ids_placeholder = "Επιλέξτε Κατηγορίες Άρθρων";
	public $APL_except_cat_ids_desc = "<strong>ΚΑΤΗΓΟΡΙΕΣ ΠΟΥ ΘΑ ΕΞΑΙΡΕΘΟΥΝ</strong><br />--------------------------------------<br /><br />Ποιες κατηγορίες θέλετε να εξαιρεθούν;<br /><br />Αυτό θα έχει ως αποτέλεσμα να παρακαμφθούν όλα τα άρθρα που ανήκουν στις κατηγορίες αυτές.<br /><br />Η επιλογή αυτή αναιρεί την κατηγορία από την παραπάνω παράμετρο (περιλαμβανόμενες κατηγορίες), εάν δοθεί και σε αυτή.";
}
?>

languages/english.php
<?php
/*
  __ _  ___  ___ ___   ___   ___     ____ _ __ ___   ___
 / _` |/  / / __/ _ \ / _ \ /  /    / __/| '_ ` _ \ /  /
| (_| |\  \| (_| (_) | (_) |\  \   | (__ | | | | | |\  \
 \__,_|/__/ \___\___/ \___/ /__/    \___\|_| |_| |_|/__/
                                                                  
*************************************************************************************
* @ Project ASCOOS                                                                  *
* @ Copyright (C) 2007 - 2012 AlexSoft Software.                                    *
* @ Address: Konstantinoupoleos 88, GR 68100, Alexandroupolis, Evros, Greece        *
* @ Tel: +30 2551 031999                                                            *
* @ Creator: Drogidis Christos                                                      *
* @ ASCOOS CMS Site: www.ascoos.com                                                 *
* @ Creator Site: www.alexsoft.gr                                                   *
* @ emails: webmaster@ascoos.com, webmaster@alexsoft.gr                             *
* @ license site: http://www.alexsoft.gr/licence/ascoos/index.php                   *
* @ Copyrighted Commercial Software                                                 *
* @ Program ASCOOS CMS Manager                                                      *
*************************************************************************************

* @ Package            : ASCOOS CMS - Frontend
* @ Subpackage         : Block Manager - Popular Contents 
* @ ASCOOS Version     : Lite - 1.0.0
* @ File Name          : /blocks/popular_conent/languages/greek.php
* @ File No.           : 7 - $release: 1.0 - $revision: 0 - $build: 0
* @ Created Date       : 2012-07-01 20:00:00 UTC+2 
* @ Updated Date       : 
* @ Author             : Drogidis Christos
* @ Author email       : webmaster@alexsoft.gr
* @ Author website     : www.alexsoft.gr
* @ Translator         : Kourtidis George
* @ Translator email   : 
* @ Translator website : 
* @ Translate Date     : 2012-07-01 20:00:00 UTC+2 
*/

defined ("ALEXSOFT_RUN_CMS") or die("Prohibition of Access.");

class TBlock_popular_content_Language extends TObject
{
	public $blockname = "Popular Content";

	public $APL_license = "Ascoos General License (AGL)";
	public $APL_author = "DROGIDIS CHRISTOS";
	public $APL_creation = "";
	public $APL_copyright = "ALEXSOFT SOFTWARE";
	public $APL_translator = "KOURTIDIS GEORGE";
	public $APL_translatorEmail = "";
	public $APL_translatorUrl = "";
	public $APL_desc = "Displays a box with the most popular site contents.";

	public $APL_paramgroup_general_label = "▼ &nbsp; General Configuration Options";
	public $APL_paramgroup_general_label_info = "<strong>GENERAL OPTIONS PARAMETERS</strong><br />--------------------------------------<br /><br />In this context you can choose from several general parameters and dynamically configure the block";
	public $APL_paramgroup_categories_label = "▼ &nbsp; Options Classes";
	public $APL_paramgroup_categories_label_info = "<strong>CHOICES OF CATEGORIES</strong><br />--------------------------------------<br /><br />In this frame you can shape the parameters of origin of content of block";
	public $APL_count_label = "Number of articles";
	public $APL_count_desc = "<strong>NUMBER OF ARTICLES</strong><br />--------------------------------------<br /><br />Here you must select the number of articles that will appear";
	public $APL_all_lang_label = "Showing regardless of language";
	public $APL_all_lang_desc = "<strong>APPEARANCE INDEPENDENT BY LANGUAGE OF SYNTAX</strong><br />--------------------------------------<br /><br />Want to include all articles, independent from the language that was drawn up?<br /><br />This will result to display all articles regardless of language syntax.<br /><br />For more information, on the left, will display a flag to specify the language syntax.";
	public $APL_show_hits_label = "Appearance of Projections";
	public $APL_show_hits_desc = "<strong>APPEARANCE OF PROJECTIONS</strong><br />--------------------------------------<br /><br />Want to presented the column with the total projections of content?";
	public $APL_theme_label = "Theme of appearance of Block";
	public $APL_theme_placeholder = "Select theme of appearance of block";
	public $APL_theme_desc = "<strong>THEME OF APPEARANCE OF BLOCK</strong><br />--------------------------------------<br /><br />Select the theme that will be implemented block &laquo;%s&raquo;.<br /><br />Each block has at least a predetermined theme, called &laquo;Default&raquo;";
	
	public $APL_type_label = "Types of Articles";
	public $APL_type_placeholder = "Select types of articles";
	public $APL_type_desc = "<strong>TYPES OF ARTICLES</strong><br />--------------------------------------<br /><br />What types of articles you want included? This will result in the appearance only of the articles are as a type of defined.";
	public $APL_cat_ids_label = "Categories that will be included";
	public $APL_cat_ids_placeholder = "Select Categories Articles";
	public $APL_cat_ids_desc = "<strong>CATEGORIES THAT WILL BE INCLUDED</strong><br />--------------------------------------<br /><br />Which categories do you want to include?<br /><br />This will result in only the articles have one of these categories appear in the list of recent articles.";
	public $APL_except_cat_ids_label = "Categories to be excluded";
	public $APL_except_cat_ids_placeholder = "Select article categories";
	public $APL_except_cat_ids_desc = "<strong>CATEGORIES TO BE EXCLUDED</strong><br />--------------------------------------<br /><br />Which categories you want to exclude?<br /><br />This will lead to sidestep all the articles belong to these categories.<br /><br />This choice reverses the category from the above parameter (included categories), if it is also given in this.";
}
?>